pkg/bpf/collection.go

```diff
@@ -41,6 +48,135 @@ func LoadCollectionSpec(path string) (*ebpf.CollectionSpec, error) {
 	return spec, nil
 }
 
+func RemoveUnreachableTailcalls(spec *ebpf.CollectionSpec) error {
+	return removeUnreachableTailcalls(spec)
```

Comment:
Why the unexported function?

Reply:
Ah, that was temporary while testing from outside the package, it can go now.